packages/babel-helpers/src/helpers/applyDecs2305.ts

```diff
@@ -43,7 +43,8 @@ type DecoratorInfo =
     ]
   | [classDecs: Function[]];
 
-/**
+function applyDecs2305Factory() {
```

Comment:
Is this necessary? I remember I moved all the top level functions for better minimization.

Reply:
I am not familiar with the minification part, currently we use factories for other deprecated decorator transforms as well. If they are unnecessary I can open a new PR to remove all `applyDecs` factories.

Reply:
I remember we used factory functions to avoid conflicts between functions with the same name in the top-level scope. In version 2305, I moved all top-level functions to the function body, so it should be able to avoid factory functions, while keeping the old version as is would be fine.

Reply:
Thanks! In that case I can revert the `2305` changes.